repoSets.json

```diff
@@ -210,6 +210,7 @@
         "aspnet/Routing",
         "aspnet/IdentityService",
         "aspnet/Testing",
+        "aspnet/Templating",
```

Comment:
I think this repo is in the next section below as well (Assigned only), so if you want it here, remove it from there.

Reply:
Removed